internal/app/cli/usage.go

```diff
@@ -0,0 +1,17 @@
+package cli
+
+import (
+	"fmt"
+	"io"
+	"os"
+)
+
+func WriteUsage(w io.Writer) {
+	fmt.Fprintf(w, "Ktor is a tool mainly for generating Ktor projects.\n\n")
```

Comment:
@vnikolova, could you please check the readme?

Reply:
Is there supposed to be a README file for how to run the project locally as well?

Reply:
Not yet